data/database.py

```diff
@@ -1426,6 +1428,7 @@ class RepositoryNotification(BaseModel):
     config_json = TextField()
     event_config_json = TextField(default="{}")
     number_of_failures = IntegerField(default=0)
+    last_ran_ms = BigIntegerField(null=True)
```

Comment:
You'll want an index on this field if you're going to be looking up by `last_ran_ms`